database/migrations/sql/20180918_18_make_user_details_permission_restricted.sql

```diff
@@ -0,0 +1,21 @@
+/*
+ * Make user_details-child:write restricted access
+ * for creating new visitors on the visitor app
+ */
+
+ UPDATE access_role_permission
```

Comment:
This needs to be an insert, permissions are queried either on "full" or "restricted", not "full" + "restricted".

Reply:
Was that meant to say 'or'?

If not I'm unsure what you mean.